common/source/docs/common-remoteid.rst

```diff
@@ -26,17 +26,36 @@ Stand-alone devices:
 OpenDroneID
 ===========
 
+.. image:: ../../../images/remoteid.jpg
+    :target: ../../_images/remoteid.jpg
+
+
 ArduPilot provides support for OpenDroneID in firmware versions 4.2 and later via MAVLink to an OpenDroneID compatible module attached to either a serial port or DroneCAN port on the autopilot.
 
 In order to provide tamper protection, a special autopilot firmware must be built and loaded on the autopilot which: enables this feature, prevents loading non-compliant firmware via GCS or normal uploaders, and provides further tamper-proofing.
 
-An example of the required modifications for an OpenDroneID firmware version for an autopilot is shown for the `Cube Orange here <https://github.com/ArduPilot/ardupilot/tree/master/libraries/AP_HAL_ChibiOS/hwdef/CubeOrange-ODID>`__
+For details on building firmware with OpenDroneID feature, see :ref:`dev:opendroneid` for details.
+
+ArduPilot also has developed firmware for  ESP32-S3 and C3 development boards for an OpenDroneID compatible ID module for attachment by serial MAVLink connection or DroneCAN to the autopilot. A binary for flashing the board, along with instructions, is `available here <https://github.com/ArduPilot/ArduRemoteID>`__ . Note, this firmware is rapidly evolving, so check for updates frequently.
```

Comment:
suggest using "Remote ID Module" every time.

Reply:
yep...will do that